go/noise/laplace_noise_test.go

```diff
@@ -172,6 +172,495 @@ func TestDeltaForThresholdLaplace(t *testing.T) {
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: math.NaN(),
+			epsilon:         0.1,
+			alpha:           0.5,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Zero epsilon",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0,
+			alpha:           0.5,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Negative epsilon",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         -1,
+			alpha:           0.5,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Infinite epsilon",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         math.Inf(1),
+			alpha:           0.5,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "NaN epsilon",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         math.NaN(),
+			alpha:           0.5,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Non-zero delta",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0.1,
+			delta:           1,
+			alpha:           0.5,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Zero alpha",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0.1,
+			alpha:           0,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Negative alpha",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0.1,
+			alpha:           -1,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "1 alpha",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0.1,
+			alpha:           1,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "Greater than 1 alpha",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0.1,
+			alpha:           2,
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+		{
+			desc:            "NaN alpha",
+			noisedX:         0,
+			l0Sensitivity:   1,
+			lInfSensitivity: 1,
+			epsilon:         0.1,
+			alpha:           math.NaN(),
+			want:            ConfidenceInterval{},
+			wantErr:         true,
+		},
+	} {
+		got, err := lap.ComputeConfidenceIntervalFloat64(tc.noisedX, tc.l0Sensitivity, tc.lInfSensitivity,
+			tc.epsilon, tc.delta, tc.alpha)
+		if (err != nil) != tc.wantErr {
+			t.Errorf("ComputeConfidenceIntervalFloat64: when %v for err got %v, want %t", tc.desc, err, tc.wantErr)
+			continue
```

Comment:
Does this `continue` do anything? If not, we should remove it.

Reply:
The idea is if you are getting an error when you are not expecting one, you shouldn't produce an t.Errorf for lower and upper bounds not being equal to the needed confidence interval in the tests logs. Tests logs were more readable this way.